### ISSUE TITLE
Sidebar Collapse Button is not working after upgrading

### ISSUE THREAD

**Issue by claytongf** [jeroennoten/Laravel-AdminLTE#629]
I've upgraded the package from 3.2 to 3.3. After upgrading, the buttons from left and right sidebars are not working properly.
The button from left sidebar does not work (does not open or close the sidebar)
And the button from the right sidebar opens and close right after clicking.

I tried to check in console and no errors found. Any ideas?

**Comment by claytongf** [comment]
I've found a problem.
The problem is when using the Laravel Mix.
When adding the line in bootstrap.js
require('../../vendor/almasaeed2010/adminlte/dist/js/adminlte');
is conflicting with the views which already imported the adminlte.min.js
<script src="{{ asset('vendor/adminlte/dist/js/adminlte.min.js') }}"></script>

If I comment one of these, the menu works properly.
The question is: Are they the same file? Or is there another solution for this case?

**Comment by Shidersz** [comment]
Please, can you try the next solution:

**1.** Edit the `master.blade.php`  file, so the `<body>` section looks like this:

```blade
<body class="@yield('classes_body')" @yield('body_data')>

    {{-- Body Content --}}
    @yield('body')

    {{-- Base Scripts --}}
    @if(!config('adminlte.enabled_laravel_mix'))
        <script src="{{ asset('vendor/jquery/jquery.min.js') }}"></script>
        <script src="{{ asset('vendor/bootstrap/js/bootstrap.bundle.min.js') }}"></script>
        <script src="{{ asset('vendor/overlayScrollbars/js/jquery.overlayScrollbars.min.js') }}"></script>

        {{-- Configured Scripts --}}
        @include('adminlte::plugins', ['type' => 'js'])

        {{-- NEXT LINE IS ADDED --}}
        <script src="{{ asset('vendor/adminlte/dist/js/adminlte.min.js') }}"></script>
    @else
        <script src="{{ mix(config('adminlte.laravel_mix_js_path', 'js/app.js')) }}"></script>
    @endif

    {{-- Custom Scripts --}}
    @yield('adminlte_js')

</body>
```
**2.** On all the blade files that `@extends('adminlte::master')` change the next lines of code:

```blade
@section('adminlte_js')
    <script src="{{ asset('vendor/adminlte/dist/js/adminlte.min.js') }}"></script>
    @stack('js')
    @yield('js')
@stop
```
by the next ones:

```blade
@section('adminlte_js')
    @stack('js')
    @yield('js')
@stop
```
Those files should be:

- auth/passwords/confirm.blade.php
- auth/auth-page.blade.php
- page.blade.php

Do those changes fixes the problem?

**Comment by lacodimizer** [comment]
The fix is released with v3.4.0.
